### ISSUE TITLE
Converter does not render all block refs properly

### ISSUE THREAD

**Issue by shugenja** [RoamJS/workbench#359]
Converter does not properly render block ref into text content, but rather retains the UID. The scenario in which this problem occurs, however, is specific. See the screenshots below:

<img width="340" alt="Screen Shot 2021-03-14 at 12 34 34 PM" src="https://user-images.githubusercontent.com/71242954/111090980-e77e2000-8507-11eb-939a-bc1761c2c071.png">

Results in the following:

<img width="556" alt="Screen Shot 2021-03-14 at 12 35 17 PM" src="https://user-images.githubusercontent.com/71242954/111090988-ebaa3d80-8507-11eb-9cef-25e3d7ed860a.png">

(Submitted at the request of RoamHacker on Academia Roamana.)

**Comment by dvargasfuertes** [comment]
Roam42 Issue Log Audit - Is this still an issue?

**Comment by shugenja** [comment]
Yes, it still happens it seems. I've attached what might be a clearer image of the problem. 

<img width="174" alt="Screen Shot 2021-06-14 at 7 27 45 PM" src="https://user-images.githubusercontent.com/71242954/121971385-9b30bf00-cd46-11eb-9b4e-24e7dc8694f0.png">

The result when put through the converter:

<img width="157" alt="Screen Shot 2021-06-14 at 7 28 32 PM" src="https://user-images.githubusercontent.com/71242954/121971432-b6033380-cd46-11eb-882a-27d77372a305.png">
